packages/gatsby/src/utils/webpack.config.js

```diff
@@ -29,6 +30,18 @@ module.exports = async (
 ) => {
   const directoryPath = withBasePath(directory)
 
+  // Polyfill for IE support
+  const supportedBrowsers = browserslist(program.browserslist)
+  let iePolyfill = false
+  if (
+    supportedBrowsers.includes(`ie 9`) ||
+    supportedBrowsers.includes(`ie 10`)
+  ) {
+    iePolyfill = `react-app-polyfill/ie9`
```

Comment:
Actually we only need to polyfill `Set` `Map` `Symbol` and other polyfills will be done by `babel-loader`, we could create our own polyfill rules instead of using `react-app-polyfill`, but I love the best practice from `create-react-app`